constructors/bq/doc/REFERENCE.md

```diff
@@ -4,6 +4,46 @@
 
 This module contains functions that create geographies from coordinates or already existing geographies.
 
+### ST_BEZIERSPLINE
+
+{{% bannerNote type="code" %}}
+constructors.ST_BEZIERSPLINE(geog, sharpness)
```

Comment:
Missing `resolution`

Reply:
True, I added this parameters a bit after and missed to update the REFERENCE. Now should be good